app/assets/stylesheets/application.css

```diff
@@ -34,6 +34,11 @@ body {
   background: #C8D5B9;
 }
 
+.section {
```

Comment:
I'm gonna merge this in but there may be a way to do this with built in bootstrap css! https://getbootstrap.com/docs/4.1/utilities/spacing/